src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/index.tsx

```diff
@@ -241,6 +255,15 @@ class DebugImageDetails extends AsyncComponent<Props, State> {
               <Label coloredBg>{t('Architecture')}</Label>
               <Value coloredBg>{architecture ?? <NotAvailable />}</Value>
             </GeneralInfo>
+            {debugFilesSettingsLink && (
+              <SearchInSettingsAction>
+                <Tooltip title={t('Search for this debug file in settings')}>
```

Comment:
```suggestion
                <Tooltip title={t('Search for this debug file in all images for the %s project', projectId)}>
```

🤷 This might be clearer to new/unfamiliar users.